scripts/generate-PR-body.sh

```diff
@@ -0,0 +1,24 @@
+echo "pr_title=Sync with ethereum/solidity@develop $(date +%d-%m-%Y)" >> $GITHUB_ENV
+
+PR_body=$(cat << EOF
+This PR was automatically generated. 
+
+Merge changes from [solidity](https://github.com/ethereum/solidity)@develop 
+The following files have conflicts and may need new translations: 
+EOF
+)
+
+echo "pr_body<<EOF"                                             >> $GITHUB_ENV
+echo "$PR_body"                                                 >> $GITHUB_ENV
+git status --short | grep "docs" | \
+awk '{print "- [ ] ["$2"]\
+(https://github.com/ethereum/solidity/tree/develop/"$2")"}'     >> $GITHUB_ENV
+echo ""                                                         >> $GITHUB_ENV
+echo "Please fix the conflicts by pushing new commits \
+to this pull request, either by editing the files \
+directly on GitHub or by checking out this branch."             >> $GITHUB_ENV
+echo ""                                                         >> $GITHUB_ENV
+echo "## DO NOT SQUASH MERGE THIS PULL REQUEST!"                >> $GITHUB_ENV
+echo "Doing so will "erase" the commits from main \
+and cause them to show up as conflicts the next time we merge." >> $GITHUB_ENV
```

Comment:
for some reason it also worked https://github.com/solidity-docs-test/pl-polish/pull/19

Reply:
It didn't. Note that the quotes around `erase` don't show up in the PR body. It's because the "inner" quotes are not really inside. Bash sees the argument as two quoted strings on both sides of `erase` rather than as a single string with a quoted word inside :)

Reply:
yeah, I did not even notice missing quotation marks.